crates/matrix-sdk-base/src/sliding_sync.rs

```diff
@@ -60,8 +64,8 @@ impl BaseClient {
         // We declare default values that can be referenced hereinbelow. When we try to
         // extract values from `e2ee`, that would be unfortunate to clone the
         // value just to pass them (to remove them `e2ee`) as a reference later.
-        let device_one_time_keys_count = Default::default();
-        let device_unused_fallback_key_types = Default::default();
+        let device_one_time_keys_count = BTreeMap::<DeviceKeyAlgorithm, UInt>::default();
```

Comment:
I just meant the `BTreeMap`, not `::<_, _>`. But I guess it doesn't hurt either.